definitions/procedures/content/prepare.rb

```diff
@@ -0,0 +1,15 @@
+module Procedures::Content
+  class Prepare < ForemanMaintain::Procedure
+    metadata do
+      description 'Prepare content for Pulp 3'
+    end
+
+    def run
+      if feature(:pulp3)
```

Comment:
Instead of adding condition here, you can set `for_feature` inside metadata. Example -

```
metadata do
      description 'Prepare content for Pulp 3' 
      for_feature :pulp3
end
```

So that this procedure will be available only when `pulp3` feature exists.

Reply:
Thanks, updated!